app/src/main/java/org/openimis/imispolicies/CheckCommission.java

```diff
@@ -564,8 +568,7 @@ public void run() {
 
 
     public void LoginDialogBox() {
-
-        //final int[] userid = {0};
+        ca.CheckInternetAvailable();
```

Comment:
If there is no connectivity, it will not stop the function.

Reply:
Fixed